cnf-certification-test/tnf_config.yml

```diff
@@ -36,6 +36,6 @@ servicesignorelist:
   - "new-pro-controller-manager-metrics-service"
   - "mysql"
 collectorAppEndPoint: ""
-executedBy: "default-executed-by"
+executedBy: ""
```

Comment:
Are these fields necessary to add with empty string?

Reply:
The partners are supposed to fill them.

Reply:
Yes, but this is the config for CI, so these fields can be empty. ExecutedBy is the mandatory, but the others not. So, we do not need those fields to be present in the config for CI test.

Reply:
Though the tnf_config.yml file is not only for CI use, so won't it be more convenient for them to be there - so the partners just have to fill them in if they want.

Reply:
and we should probably also add the ec2 endpoint as default for the `collectorAppEndPoint` in the tnf-config

Reply:
Partners dont need to fill ec2 endpoint. The code that parses this tnf_config should fill the collector_url and make request to there based on executedBy.

Reply:
why based on executedBy? we run the CI testin rds too.. So I guess it's always ec2's endpoint. 
Though I agree it shouldn't be set in the tnf_config .

Reply:
IIRC, this field was mandatory for claim upload, the rest is not.

Reply:
Ah, you are replacing these values. Looks good then!